types/collectionsjs/index.d.ts

```diff
@@ -0,0 +1,46 @@
+// Type definitions for collectionsjs 0.3
+// Project: https://github.com/logaretm/collectionsjs#readme
+// Definitions by: Jaymeh <https://github.com/jaymeh>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.1
+
+export default class Collection {
```

Comment:
This seems like a prime candidate for a generic eg. `export default class Collection<T>` and replace all(where applicable) usage of `any` with T.

Reply:
Thanks for pointing this out, I'm rather new to typescript so this concept is a little confusing but I've now revised my request, hopefully correctly. 👍